docs/installation.rst

```diff
@@ -41,3 +41,12 @@ context processor:
         ...
         'django.template.context_processors.request',
     )
+
+
+WSGI + Nginx
+#####
```

Comment:
Some rST validators require this line to be as long as the title length (in ASCII characters). Let's see what the GitHub CI says, I'm personally fine with either formatting :)